core-api/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
   testCompile group: 'org.springframework.boot',  name: 'spring-boot-starter',            version: springBootVersion
   testCompile group: 'org.springframework.boot',  name: 'spring-boot-starter-test',       version: springBootVersion
   testCompile group: 'junit',                     name: 'junit',                          version: junitVersion
+  testCompile group: 'org.glassfish.web',         name: 'javax.el',                       version: '2.2.6'
```

Comment:
do we using this one ?

Reply:
yes, this dep provides the expression language implementation to be able to activate the annotation based validation of the related fields

Reply:
I think tomcat also has an implementation for javax.el.
https://mvnrepository.com/artifact/org.apache.tomcat/tomcat-el-api
Maybe worth to try it out, so we won't depend on glassfish.

Reply:
I'll give it a shot, thank you!